<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RemoteCreateEditDlg</class>
 <widget class="QDialog" name="RemoteCreateEditDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>428</width>
    <height>307</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <item row="0" column="0">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Remote Server</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QLabel" name="label_2">
        <property name="text">
         <string>Url</string>
        </property>
        <property name="buddy">
         <cstring>txtUrl</cstring>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="LineEdit" name="txtUrl">
        <property name="mandatory" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Name</string>
        </property>
        <property name="buddy">
         <cstring>txtName</cstring>
        </property>
       </widget>
      </item>
      <item row="0" column="1">
       <widget class="LineEdit" name="txtName">
        <property name="mandatory" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QCheckBox" name="chkPushUrl">
        <property name="text">
         <string>Push Url</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="LineEdit" name="txtPushUrl">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="mandatory" stdset="0">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QGroupBox" name="chkRefSpecs">
     <property name="title">
      <string>Custom refspec for Fetch</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="1">
       <widget class="QPushButton" name="btnAddRefSpec">
        <property name="text">
         <string>Add</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLineEdit" name="txtEditRefSpec"/>
      </item>
      <item row="0" column="0" rowspan="4">
       <widget class="QTreeView" name="treeRefSpecs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </item>
      <item row="2" column="1">
       <widget class="QPushButton" name="btnRemoveRefSpec">
        <property name="text">
         <string>Remove</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="QPushButton" name="btnUpdateRefSpec">
        <property name="text">
         <string>Update</string>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <spacer name="verticalSpacer">
        <property name="orientation">
         <enum>Qt::Vertical</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>20</width>
          <height>40</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>LineEdit</class>
   <extends>QLineEdit</extends>
   <header>Widgets/LineEdit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtName</tabstop>
  <tabstop>txtUrl</tabstop>
  <tabstop>chkPushUrl</tabstop>
  <tabstop>txtPushUrl</tabstop>
  <tabstop>chkRefSpecs</tabstop>
  <tabstop>treeRefSpecs</tabstop>
  <tabstop>txtEditRefSpec</tabstop>
  <tabstop>btnAddRefSpec</tabstop>
  <tabstop>btnUpdateRefSpec</tabstop>
  <tabstop>btnRemoveRefSpec</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RemoteCreateEditDlg</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>252</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RemoteCreateEditDlg</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>320</x>
     <y>302</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>chkPushUrl</sender>
   <signal>toggled(bool)</signal>
   <receiver>txtPushUrl</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>103</y>
    </hint>
    <hint type="destinationlabel">
     <x>106</x>
     <y>99</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
